frontends/ol-components/src/components/TruncateText/TruncateText.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from "@emotion/react"
+import styled from "@emotion/styled"
+
+type TruncateTextProps = {
+  /**
+   * Number of lines to display before truncating text.
+   */
+  lineClamp?: number | "none"
+}
+
+const truncateText = (lines?: number | "none") =>
+  css({
+    display: "-webkit-box",
+    WebkitBoxOrient: "vertical",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    WebkitLineClamp: lines,
```

Comment:
This says webkit, but works in all slightly modern browsers beyond IE. This will not work in IE, but will fallback should be somewhat graceful—no text truncation, all text visible. This seems acceptable.

In contrast, the behavior of `react-dotdotdot` with our font + line height does not fail gracefully: when set to `clamp={x}`, it seems generally to clamp `x-1`. In particular, that means clamping to 1 line clamps to zero.